encoder/src/main/java/com/pedro/encoder/input/video/Camera1ApiManager.java

```diff
@@ -63,23 +64,26 @@
 
   private FaceDetectorCallback faceDetectorCallback;
 
-  public Camera1ApiManager(SurfaceView surfaceView, GetCameraData getCameraData) {
+  public Camera1ApiManager(SurfaceView surfaceView, GetCameraData getCameraData, CameraSwitchCallback cameraSwitchCallback) {
     this.surfaceView = surfaceView;
     this.getCameraData = getCameraData;
     this.context = surfaceView.getContext();
+    this.cameraSwitchCallback = cameraSwitchCallback;
     init();
   }
 
-  public Camera1ApiManager(TextureView textureView, GetCameraData getCameraData) {
+  public Camera1ApiManager(TextureView textureView, GetCameraData getCameraData, CameraSwitchCallback cameraSwitchCallback) {
     this.textureView = textureView;
     this.getCameraData = getCameraData;
     this.context = textureView.getContext();
+    this.cameraSwitchCallback = cameraSwitchCallback;
     init();
   }
 
-  public Camera1ApiManager(SurfaceTexture surfaceTexture, Context context) {
+  public Camera1ApiManager(SurfaceTexture surfaceTexture, Context context, CameraSwitchCallback cameraSwitchCallback) {
     this.surfaceTexture = surfaceTexture;
     this.context = context;
+    this.cameraSwitchCallback = cameraSwitchCallback;
```

Comment:
Do we need this in constructor? I guess it should be optional

Reply:
Removed